### PR TITLE
Collect pxb and stork logs for all the tests

### PR DESCRIPTION
**Pull request by sn-px**
**What this PR does / why we need it**:
Collect pxb and stork logs for all the tests irrespective of PASS or FAIL

Currectly. logs are getting collected only if the testcase fails
Logs do not get collected when the failure is seen in justaftereach

**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PB-5216

**Special notes for your reviewer**:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/3882/consoleFull
Screenshot 2023-12-20 at 12 31 00 PM

### CODE REVIEW

File: tests/common.go [hunk 1]
```diff
@@ -7180,24 +7180,22 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
 		AfterEachTest(contexts, CurrentTestRailTestCaseId, RunIdForSuite)
 	}
 	ginkgoTestDescr := ginkgo.CurrentGinkgoTestDescription()
-	if ginkgoTestDescr.Failed {
```
Comment (kphalgun-px) [inline tests/common.go:7183]: How about?

```go
if ginkgoTestDescr.Failed {
    log.Infof(">>>> FAILED TEST: %s", ginkgoTestDescr.FullTestText)
}
log.Infof(">>>> Collecting logs for testcase : %s", ginkgoTestDescr.FullTestText)
.
.
.
```

Comment (sn-px) [reply tests/common.go:7183]: we don't want to differentiate between test which passed or failed , one string "Collecting logs for testcase" should be good enough

File: tests/common.go [hunk 2]
```diff
@@ -7180,24 +7180,22 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
...
-		log.Infof(">>>> FAILED TEST: %s", ginkgoTestDescr.FullTestText)
```
Comment (mkoppal-px) [inline tests/common.go:7184]: Let us retain this log message `log.Infof(">>>> FAILED TEST: %s", ginkgoTestDescr.FullTestText)`

File: tests/common.go (hunk 1 above)
Comment (sn-px) [reply tests/common.go:7183]: done retained the FAILED TEST comment

File: tests/common.go (hunk 2 above)
Comment (sn-px) [reply tests/common.go:7184]: retained the failed test comment

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by kphalgun-px — APPROVED** [review]
LGTM

**Pull request merged**